internal/pkg/cli/app_init.go

```diff
@@ -22,6 +22,19 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const (
+	fmtAppInitAppTypePrompt  = "Which type of %s best represents your application?"
```

Comment:
Do you think it'd be easier to read if we made these vars and inline the full prompt string value?

For example:
```go
var fmtAppInitAppTypePrompt = fmt.Sprintf("Which type of %s best represents your application?", color.Emphasize("infrastructure pattern"))
```

I find it distracting having to hunt for where this value is used to see what we're replacing the `"%s"` with.

Reply:
Done!